cli/operator/node.go

```diff
@@ -679,135 +696,85 @@ func setupConsensusClient(
 	return cl
 }
 
-func setupEventHandling(
+func startMetricsHandler(ctx context.Context, logger *zap.Logger, db basedb.Database, metricsReporter metricsreporter.MetricsReporter, port int, enableProf bool) {
+	logger = logger.Named(logging.NameMetricsHandler)
+	// init and start HTTP handler
+	metricsHandler := metrics.NewMetricsHandler(ctx, db, metricsReporter, enableProf, operatorNode.(metrics.HealthChecker))
+	addr := fmt.Sprintf(":%d", port)
+	if err := metricsHandler.Start(logger, http.NewServeMux(), addr); err != nil {
+		logger.Panic("failed to serve metrics", zap.Error(err))
```

Comment:
Why do you prefer `Panic` over `Fatal` here?

Reply:
Honestly - no idea, I guess it's just an old code,

and we have 4 `logger.Panic` usages in the codebase in total too, changed to `Fatal ` everywhere.